hasura/README.md

```diff
@@ -13,9 +13,9 @@ and all of our backend services.
 
 1. `cp hasura/.env.example hasura/.env.local`
 1. In `hasura/.env.local` set the value of `HASURA_GRAPHQL_ADMIN_SECRET` to the Hasura admin secret value generated at the beginning of this setup process. Set the value of `EVENT_SECRET` to another secure random value.
-1. If you are using an M1 Mac, note that Hasura's official Docker image does not yet support this platform. You will need to use a different image built by a third party. Swap `image: hasura/graphql-engine:v{version number}.cli-migrations-v2` for `image: fedormelexin/graphql-engine-arm64:v{version number}.cli-migrations-v2` in `docker-compose.yaml`, replacing the version number as appropriate.
 1. Run the `Hasura Console -- Local Development` task within VSCode:
    `Terminal > Run Task`. This will start the GraphQL engine in Docker first, and that VSCode task will prompt for a Docker project name with a default filled in. Press enter to accept the default.
+   - Due to a race condition between writing Hasura's metadata and applying migrations, we recommend running these sequentially. To do this, comment out the `volume` `./hasura/metadata:/hasura-metada` in `docker-compose.yaml` and run the `Hasura Console -- Local Development` task. Once complete, uncomment `./hasura/metadata:/hasura-metada` and run `Hasura Console -- Local Development` again.
```

Comment:
We don't absolutely know the underlying cause here, so maybe say something like 'If you find that the Docker container runs out of memory or crashes on first startup, then...'.